src/Controller/Component/RequestHandlerComponent.php

```diff
@@ -92,10 +86,6 @@ public function __construct(ComponentRegistry $registry, array $config = [])
                 'xml' => 'Xml',
                 'ajax' => 'Ajax',
             ],
-            'inputTypeMap' => [
```

Comment:
This wasn't emitting deprecations in 3.x removing it now could end up breaking applications without much warning.

Reply:
That's an unfortunate oversight. I have added deprecation warning in `startup()` which is shown if input data is populated but request data/parsed body isn't.